tests/integration/test_multi_objective_bayesian_optimization.py

```diff
@@ -31,13 +34,34 @@
 
 @random_seed
 @pytest.mark.parametrize(
-    "num_steps, acquisition_rule",
+    "num_steps, acquisition_rule, convergence_threshold",
     [
-        (20, EfficientGlobalOptimization(ExpectedHypervolumeImprovement().using(OBJECTIVE))),
+        pytest.param(
```

Comment:
What does this pytest.param do?

Reply:
am using it more for an id printing purpose, e.g., `ehvi_vlmop2` printed during testing maybe more interpretable as telling what the test is for than `20-acquisition_rule0--3.65` when having too many tests.